pennylane/transforms/qfunc_transforms.py

```diff
@@ -65,30 +66,14 @@ def qfunc(x):
     """
 
     def wrapper(*args, **kwargs):
-        active_tape = qml.QueuingManager.active_context()
-        new_tape = qml.tape.QuantumTape() if active_tape is None else active_tape.__class__()
-        with qml.QueuingManager.stop_recording(), new_tape:
+        with qml.QueuingManager.stop_recording(), qml.tape.QuantumTape() as new_tape:

```

Comment:
Do we want to deprecate `qml.transforms.make_tape` soon?

Reply:
yes : ) it should have only been used internally anyway, and I have removed all internal references. I'll make a story to ensure we deprecate it in a subsequent PR.